sov-modules/sov-modules-impl/accounts/src/call.rs

```diff
@@ -0,0 +1,80 @@
+use crate::Account;
+use crate::Accounts;
+use anyhow::Result;
+use borsh::{BorshDeserialize, BorshSerialize};
+use sov_modules_api::Signature;
+use sov_modules_api::{Address, CallResponse, PublicKey};
+
+pub const UPDATE_ACCOUNT_MSG: [u8; 32] = [0; 32];
```

Comment:
(Question) Can setting the message to all zeros interacts poorly with signature schemes since they interpret messages as numbers?

Reply:
This has to be handled by signature schema itself. 
1. Messages are handled as numbers but [0;32] message does not necessarily maps to `0`  point. It can be that `[27;32]` means 0 for a particular schema so we can't really defend agains it. 
2. BLS for example has this property that `0` private key can produce signatures valid for any message and this is handled on the verification part.
3. We can (very unlikely but still) can generate a valid message that hashes to 0 and the signature schema should work.

But now, I think maybe we want our `burn` address to be `[0;32]` so it would be good to use a different number here (just to avoid confusion). I will change it to [1;32]